stock_manual_transfer/views/stock_manual_transfer_views.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="stock_manual_transfer_form_view" model="ir.ui.view">
+        <field name="name">stock.manual_transfer.form</field>
+        <field name="model">stock.manual_transfer</field>
+        <field name="arch" type="xml">
+            <form string="Manual Transfer">
+                <header>
+                    <field name="picking_ids" invisible="1" />
+                    <button
+                        name="action_validate"
+                        type="object"
+                        string="Make transfers"
+                        invisible="state != 'draft'"
+                        class="oe_highlight"
+                        groups="stock_manual_transfer.group_stock_manual_transfer"
+                    />
+                    <button
+                        name="action_view_pickings"
+                        type="object"
+                        string="View Transfers"
+                        invisible="not picking_ids"
+                        class="oe_highlight"
+                        groups="stock_manual_transfer.group_stock_manual_transfer"
+                    />
+                    <field name="state" widget="statusbar" />
+                </header>
+                <sheet>
+                    <group>
+                        <h1>
+                            <field name="name" force_save="1" />
+                        </h1>
+                    </group>
+                    <group col="4">
+                        <field name="warehouse_id" options="{'no_create': True}" readonly="state != 'draft'" />
+                        <field name="date_planned" readonly="state != 'draft'" />
+                        <field name="route_id" options="{'no_create': True}" readonly="state != 'draft'" />
+                    </group>
+                    <notebook>
+                        <page string="Transfer Lines" name="transfer_lines">
+                            <field name="transfer_line_ids" nolabel="1" readonly="state != 'draft'">
+                                <tree editable="bottom">
+                                    <field name="sequence" widget="handle" />
+                                    <field name="product_id" />
+                                    <field name="product_uom_qty" />
+                                    <field name="product_uom_id" optional="show" />
```

Comment:
Why?

Reply:
Because if the user don't belong to that group the compute of that field was not triggered but I've removed it and everything works fine.

Reply:
I remember that the tests failed so I test it functionally and worked with demo user so I fix the test removing the `sudo()`.

Reply:
What is the error message?

You should probably add this same file without group but invisible